test/hocr-eval-lines/hocr-eval-lines.tsht

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env tsht
+TESTDATA="../testdata"
+
+plan 3
+
+exec_ok "hocr-eval-lines" "-v" "$TESTDATA/sample.txt" "$TESTDATA/sample.html"
+ocrerrors=($(hocr-eval-lines "$TESTDATA/sample.txt" "$TESTDATA/tess.hocr" 2>&1|grep '^ocr_errors 7.0'))
+segerrors=($(hocr-eval-lines "$TESTDATA/sample.txt" "$TESTDATA/tess.hocr" 2>&1|grep '^segmentation_errors 0'))
+equals "${#ocrerrors[@]}" 2 "hocr-eval-lines: check whether detection of ocr_errors is correct"
+equals "${#segerrors[@]}" 2 "hocr-eval-lines: check whether detection of segmentation_errors is correct"
```

Comment:
I don't know exactly why this has to be `2` but it works.. (I copied most of the lines from some other tests.)